cpp/src/parquet/encoding.cc

```diff
@@ -1214,18 +1234,15 @@ inline int PlainBooleanDecoder::DecodeArrow(
 
 int PlainBooleanDecoder::Decode(uint8_t* buffer, int max_values) {
```

Comment:
When refering to https://github.com/apache/arrow/issues/38355 . Previously we won't crack the trailing bit, now we'll . Should we keep the same syntax? @pitrou @wgtmac